.github/workflows/nightly.yml

```diff
@@ -76,6 +76,14 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
 
+      - name: Delete old nightly.* tags
+        run: |
+          old_tags=$(curl -s "https://hub.docker.com/v2/repositories/evcc/evcc/tags/?page_size=100" | jq -r '.results | map(select(.name | startswith("nightly."))) | sort_by(.last_updated) | reverse | .[1:] | .[].name')
```

Comment:
Just to be sure: is jq even available on CI?

Reply:
I tested it on my fork and yes, it was available within gitlab CI. I also tried to install `jq` with apt-get and it showed that it is already installed.

Reply:
https://github.com/marketplace/actions/install-jq -> _GitHub's own hosted runners come with a version of [jq pre-installed](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#preinstalled-software)._